src/omnibar/omnibar.ts

```diff
@@ -106,18 +111,28 @@ function showVerticalNav(): boolean {
       return true;
     }
 
-    let qs = BBAuthInterop.getCurrentUrl().split('?')[1];
+    const qs = parseQuerystring();
+    return qs.leftnav === '1';
+  }
 
-    if (qs) {
-      if (qs.indexOf('#') >= 0) {
-        qs = qs.split('#')[0];
-      }
+  return false;
+}
 
-      return qs.split('&').indexOf('leftnav=1') >= 0;
+function parseQuerystring(): Record<string, string> {
+  const queryParams: Record<string, string> = {};
+  let qs = BBAuthInterop.getCurrentUrl().split('?')[1];
+  if (qs) {
+    if (qs.indexOf('#') >= 0) {
+      qs = qs.split('#')[0];
     }
+    qs.split('&').forEach((p) => {
```

Comment:
I'd prefer to see a `for...of` loop here. It's easier to step through with a debugger and it doesn't require creating a new function.

```
for (const p of qs.split('&')) {
}
```